### PR TITLE
Add force checkout to cleanup changes from ncc

### PR DESCRIPTION
**Pull request by kunalgupta022**
After installing `@vercel/ncc` the local `package.json` and `package-lock.json` are modified during the build process of the action. This force checks out the branch and undoes changes made to package.json and package-lock.json.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -50,6 +50,8 @@ async function run() {
         } else {
             console.log("Node js module is up to date.");
         }
+        await git.pull(branch, ['-f']);
```
Comment (gps) [inline index.js]: This doesn't seem ideal. Is there no way for us to do this "locally"? Keep in mind that while this action is running, someone may have pushed more commits onto the PR branch.

/cc @Purushothaman22

Comment (Purushothaman22) [reply index.js]: This line adds no value as long as your running the CI locally for you to fetch the latest commit.  You have to do it locally.

Comment (kunalgupta022) [reply index.js]: For testing the `eof-newline` action I'm running the action locally on a private repo.
> You have to do it locally.

So a `git reset --hard` should be fine?

Comment (Purushothaman22) [reply index.js]: > So a git reset --hard should be fine?

Yes.

**Comment by kunalgupta022** [comment]
Closing as this is not a necessary feature

**Pull request closed without merging**